spark/src/test/scala/geotrellis/spark/pyramid/PyramidSpec.scala

```diff
@@ -129,5 +129,79 @@ class PyramidSpec extends FunSpec with Matchers with TestEnvironment {
         rdd = newRDD
       }
     }
+
+    it("should pyramid floating layer") {
+      val tileLayout = TileLayout(4, 4, 2, 2)
+
+      val dt1 = ZonedDateTime.of(2014, 5, 17, 4, 0, 0, 0, ZoneOffset.UTC)
+      val dt2 = ZonedDateTime.of(2014, 5, 18, 3, 0, 0, 0, ZoneOffset.UTC)
+
+      val tile1 =
+        ArrayTile(Array(
+          1, 1,  1, 1,   2, 2,  2, 2,
+          1, 1,  1, 1,   2, 2,  2, 2,
+
+          1, 1,  1, 1,   2, 2,  2, 2,
+          1, 1,  1, 1,   2, 2,  2, 2,
+
+
+          3, 3,  3, 3,   4, 4,  4, 4,
+          3, 3,  3, 3,   4, 4,  4, 4,
+
+          3, 3,  3, 3,   4, 4,  4, 4,
+          3, 3,  3, 3,   4, 4,  4, 4
+        ) , 8, 8)
+
+      val tile2 =
+        ArrayTile(Array(
+          10, 10,  10, 10,   20, 20,  20, 20,
+          10, 10,  10, 10,   20, 20,  20, 20,
+
+          10, 10,  10, 10,   20, 20,  20, 20,
+          10, 10,  10, 10,   20, 20,  20, 20,
+
+
+          30, 30,  30, 30,   40, 40,  40, 40,
+          30, 30,  30, 30,   40, 40,  40, 40,
+
+          30, 30,  30, 30,   40, 40,  40, 40,
+          30, 30,  30, 30,   40, 40,  40, 40
+        ) , 8, 8)
+
+      val rdd =
+        createSpaceTimeTileLayerRDD(
+          Seq( (tile1, dt1), (tile2, dt2) ),
+          tileLayout
+        )
+
+      val layoutScheme = ZoomedLayoutScheme(LatLng, 2)
+
+      val LayoutLevel(_, layout) = FloatingLayoutScheme(512).levelFor(LatLng.worldExtent, CellSize(0.5, 0.5))
+      val LayoutLevel(zoom, _) = layoutScheme.levelForZoom(LatLng.worldExtent, 2)
```

Comment:
Whats the point of this line ? `ZoomedLayoutScheme.levelForZoom` will always just give you the same zoomed you asked for, its the layout that is the real output of this function.